easybuild/easyconfigs/s/SOAPdenovo2/SOAPdenovo2-r240-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "MakeCp"
+
+name = 'SOAPdenovo2'
+version = 'r240'
+
+homepage = 'http://soap.genomics.org.cn/index.html'
+description = """SOAPdenovo is a novel short-read assembly method that can build a 
+ de novo draft assembly for human-sized genomes. The program is specially designed to 
+ assemble Illumina short reads. It creates new opportunities for building reference 
+ sequences and carrying out accurate analyses of unexplored genomes in a cost effective way. 
+ SOAPdenovo2 is the successor of SOAPdenovo."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-src-%(version)s.tgz']
+patches = ['SOAPdenovo2-fix-unistd-includes.patch']
+checksums = [
+    'b58fdb9b14766a122992d23dba5e91bd733c86e0062b432181aa5c1e7f052bb7',  # SOAPdenovo2-src-r240.tgz
+    '7d3c526eb4494f071a376a5b2826acad9048cbdddaf0ea2d6771dd3968497d00',  # SOAPdenovo2-fix-unistd-includes.patch
+]
+
+# parallel build is broken
+maxparallel = 1
```

Comment:
@vanzod Did you re-check whether this is still needed?

If it's still relevant, just use `parallel =1` which hardcodes it to `1`. With `maxparallel` EasyBuild will still check the number of available cores first, and then do `max(N, 1)` which will of course be `1`, so better hardcode to `1` straight away...

Reply:
Yes, it still needs to be built serially.